docs/standard/design-guidelines/choosing-between-anonymous-and-tuple.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Choosing between anonymous and tuple types"
+title: Choosing between anonymous and tuple types
 description: Learn how to decide whether to design a type as a class, or to design a type as a struct.
-ms.date: 06/24/2020
+ms.date: 06/26/2020
 ms.technology: dotnet-standard
 ---
 
 # Choosing between anonymous and tuple types
 
-As a developer, choosing the appropriate type can be scary task. Anonymous types have been available since C# 3.0, while generic <xref:System.Tuple%602?displayProperty=nameWithType> types were introduced with .NET Framework 4.0. There are more modern options that exist, such as <xref:System.ValueTuple%602?displayProperty=nameWithType> - which as the name implies, provide a value type with the flexibility of anonymous types. In this article, you'll learn when it's appropriate to choose one type over the other.
+As a developer, choosing the appropriate type involves considering a types usability, performance, and tradeoffs compared to other types. Anonymous types have been available since C# 3.0, while generic <xref:System.Tuple%602?displayProperty=nameWithType> types were introduced with .NET Framework 4.0. Since then new options have been introduced with language level support, such as <xref:System.ValueTuple%602?displayProperty=nameWithType> - which as the name implies, provide a value type with the flexibility of anonymous types. In this article, you'll learn when it's appropriate to choose one type over the other.
```

Comment:
nits:
```suggestion
Choosing the appropriate type involves considering its usability, performance, and tradeoffs compared to other types. Anonymous types have been available since C# 3.0, while generic <xref:System.Tuple%602?displayProperty=nameWithType> types were introduced with .NET Framework 4.0. Since then new options have been introduced with language level support, such as <xref:System.ValueTuple%602?displayProperty=nameWithType> - which as the name implies, provide a value type with the flexibility of anonymous types. In this article, you'll learn when it's appropriate to choose one type over the other.
```